### PR TITLE
Fixes fetching the Payara Server list in private network

### PR DESCRIPTION
**Pull request by jGauravGupta**
This is a bug fix to return the empty list of the Payara Server version in case maven central repository not accessible from the private network.

Issue reported here: https://github.com/apache/netbeans/pull/2409#issuecomment-867782698
Issue caused by: https://github.com/apache/netbeans/pull/2409

### CODE REVIEW

**Comment by adam-waldenberg** [comment]
Is this wise? Silently failing and not giving any feedback does not really help the user.

**Comment by bdrx312** [comment]
Thanks for looking into this. Would it be possible to pull the file from a local file location or from a configured or local maven repo? In netbeans, a user can customize their maven repositories to edit their repository url and add additional repositories. It may be more robust to use the configured maven repository  rather than a hardcoded one.

**Comment by jGauravGupta** [comment]
Thanks for the feedbacks, I will update the PR soon.

File: enterprise/payara.common/src/org/netbeans/modules/payara/common/wizards/Bundle.properties [hunk 1]
```diff
@@ -56,6 +56,8 @@ A11Y_HTTP_PORT_TXT=HTTP port
 LBL_InitializationTitle=Initialize
 LBL_ReadLicenseText=<html><body><span style="text-decoration: underline;">I have read and accept the license agreement... (click)</span></body></html>
 LBL_NewInstallDirCanBeUsed=<html><body>{0} does not exist but server can be downloaded and installed there.</body></html>
+LBL_UnaleToConnectPayaraMavenRepo=<html><body>Cannot establish a connection with the Payara maven repository.</body></html>
+LBL_UnaleToConnectPayaraMavenRepo_NewInstallDirCanBeUsed=<html><body>{0} does not exist, please browse the valid path. Cannot establish a connection with the Payara maven repository.</body></html>
```
Comment (bdrx312) [inline enterprise/payara.common/src/org/netbeans/modules/payara/common/wizards/Bundle.properties:60]: `Unale` looks like a possible typo. Should it be `Unable`?

**Comment by pepness** [comment]
This bug also happen when you do not have access to the internet, I went without an internet connection for a few days and I could not register any Application Server without an internet connection. 

I just tried this PR and it solves the main problem of not been able to register an App Server. But it is throwing an exception which it should not. 
NetBeans-Test_PR3018

**Comment by ebarboni** [comment]
@jGauravGupta thanks for the PR is the exception @pepness see solved ?

**Comment by jGauravGupta** [comment]
Yes @ebarboni

**Comment by pepness** [comment]
Can you rebase your branch onto master @jGauravGupta ? I get an error `java.lang.NumberFormatException: multiple points` when I run this PR.

**Comment by pepness** [comment]
I get another exception
Screenshot from 2021-07-19 14-15-24

**Comment by jGauravGupta** [comment]
After rebase with master can't build the project:

````
D:\PW\Ecosystem\netbeans\nbbuild\netbeans\harness\build.xml:150: java.io.IOException: Duplicated module org.objectweb.asm: found in D:\PW\Ecosystem\netbeans\nbbuild\netbeans\platform\core\asm-8.0.1.jar and D:\PW\Ecosystem\netbeans\nbbuild\netbeans\platform\core\asm-9.2.jar
        at org.netbeans.nbbuild.ModuleListParser.scanOneBinary(ModuleListParser.java:904)
        at org.netbeans.nbbuild.ModuleListParser.doScanBinaries(ModuleListParser.java:589)
        at org.netbeans.nbbuild.ModuleListParser.scanBinaries(ModuleListParser.java:551)
        at org.netbeans.nbbuild.ModuleListParser.<init>(ModuleListParser.java:764)
        at org.netbeans.nbbuild.ParseProjectXml.execute(ParseProjectXml.java:427)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
````

**Comment by jGauravGupta** [comment]
Hi @pepness, 

I rebased the PR with the master branch. May you retest the PR again?

**Comment by pepness** [comment]
Testing it now.

**Comment by pepness** [comment]
I confirm that there is no exception been thrown and I could register app servers without problems.

**Comment by ebarboni** [comment]
nice maybe @jGauravGupta you may squash or not  and merge

**Comment by jGauravGupta** [comment]
Thanks @pepness

**Pull request merged**